docs/additional-configs.md

```diff
@@ -256,7 +256,7 @@ The default is `false`. For more information, see the [associated issue](https:/
 
 - `enable-api-fields`: When using v1beta1 APIs, setting this field to "stable" or "beta"
 enables [beta features](#beta-features). When using v1 APIs, setting this field to "stable"
-allows only stable features, and setting it to "beta" allows only beta features.
```

Comment:
Isn't the behavior the same when we use v1beta1 API?  
` When using v1 APIs, setting this field to "stable" allows only stable features,`

Reply:
Thanks for the question! 
I am afraid that it isn't the case at the moment. For examples, 'stable' feature behaviors like resolver pass validations for `stable` `enable-api-fields` in v1beta1 but it does not in v1.